src/index.spec.js

```diff
@@ -60,6 +76,24 @@ describe('EventDrops', () => {
         ]);
     });
 
+    // it('should zoom to correct domain on zoomToDomain', () => {
```

Comment:
Will remove the code. I was trying to add a test for `zoomToDomain`, but for some reason, it would not zoom to the domain. Been trying to figure out why for some time now, but after going through the debugger I cannot see why it is not applying the zoom.